app/controllers/sessions_controller.rb

```diff
@@ -3,9 +3,10 @@ def new; end
 
   def create
     user = User.find_by email: params[:session][:email].downcase
-    
-    if user && user.authenticate(params[:session][:password])
+
+    if user &.authenticate params[:session][:password]
```

Comment:
&. nghĩa là gì nhỉ

Reply:
toán từ điều hướng an toàn , e sử dụng cho nó ngắn gọn hơn thôi ạ

Reply:
@hieunguyen061197 cụ thể thì nó hoạt động ra sao e

Reply:
safe navigation operator cho phép gọi một phương thức một cách an toàn trên object có thể là nil , nếu object k phải là nil thì nó sẽ gửi method đến object